mscore/capxml.cpp

```diff
@@ -663,8 +688,10 @@ QList<BasicDrawObj*> Capella::readCapxDrawObjectArray(XmlReader& e)
                               ol.append(o);
                               }
                         else if (tag == "transposable") {
-                              qDebug("readCapxDrawObjectArray: found transposable (skipping)");
-                              e.skipCurrentElement();
+							  MyTransposableObj* o = new MyTransposableObj(this);
+							  bdo = o; // save o to handle the "basic" tag (which sometimes follows)
+							  o->readCapx(e);
+							  ol.append(o);
```

Comment:
Wrong indent (6 spaces, not tabs)

Reply:
Yup, I made a commit, but you were faster than me.